.github/workflows/ci.yml

```diff
@@ -30,7 +22,7 @@ jobs:
           version=$(sed -E '/^version: (.*)$/!d; s//\1/' test/package.yaml)
           cat >> "$GITHUB_OUTPUT" <<EOM
           version=$version
-          tag=v$version
+          tag=example-$version
```

Comment:
This was just a long-standing test bug that I hit by actually bumping the test version.